target-platform-configuration/src/main/java/org/eclipse/tycho/target/TargetPlatformConfigurationMojo.java

```diff
@@ -183,7 +184,7 @@ public class TargetPlatformConfigurationMojo extends AbstractMojo {
      * </pre>
      */
     @Parameter(name = DefaultTargetPlatformConfigurationReader.DEPENDENCY_RESOLUTION)
-    private Object dependencyResolution; // TODO stronger typing
+    private PlexusConfiguration dependencyResolution; // TODO stronger typing
```

Comment:
I wonder what (beside `extraRequirements`) is actually allowed here? The doc just says "for example" ...